packages/blade/src/components/Chip/chipTokens.ts

```diff
@@ -216,7 +184,10 @@ const chipTextSizes = {
   },
 } as const;
 
-const chipMotionTokens = {
+const chipMotionTokens: Record<
+  'duration' | 'easing',
+  'motion.duration.xquick' | 'motion.easing.standard.effective'
```

Comment:
Use `Record<duration | easing, EasingString | DurationString>` so if later motion tokens change we know. 

Alternatively we can just omit the type def and use `const chipMotionTokens = {} as const` since getIn is now typesafe.

Reply:
Done!!!